client/api.go

```diff
@@ -837,7 +844,7 @@ func (c *Client) TestSendAnonVotes(
 	if err != nil {
 		return 0, err
 	}
-	log.Infof("CircuitIndex: %v, CircuitConfit: %+v", *circuitIndex, *circuitConfig)
+	log.Infof("CircuitIndex: %v, CircuitPath: %+v", *circuitIndex, *&circuitConfig.CircuitPath)
```

Comment:
Why `*&circuitConfig.CircuitPath` and not `circuitConfig.CircuitPath`? as `circuit.Config.CircuitPath is a `string`.

```suggestion
	log.Infof("CircuitIndex: %v, CircuitPath: %+v", *circuitIndex, circuitConfig.CircuitPath)
```

Reply:
Thas vs-code autocomplete, I will change it, thanks